types/box-intersect/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for box-intersect 1.0
+// Project: https://github.com/mikolalysenko/box-intersect
+// Definitions by: Sankaku <https://github.com/sankaku-deltalab>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default boxIntersect;
```

Comment:
It seems like the [underlying library](https://unpkg.com/box-intersect@1.0.2/index.js) is not using `export default`, so this .d.ts file should not either. Instead, you should follow the appropriate template from [the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/templates.html). Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.